autonomi/src/client/mod.rs

```diff
@@ -81,13 +80,11 @@ pub use ant_protocol::CLOSE_GROUP_SIZE;
 #[derive(Clone)]
 pub struct Client {
     pub(crate) network: Network,
-    pub(crate) client_event_sender: Arc<Option<mpsc::Sender<ClientEvent>>>,
+    pub(crate) client_event_sender: Option<mpsc::Sender<ClientEvent>>,
     /// The EVM network to use for the client.
-    evm_network: Arc<EvmNetwork>,
+    evm_network: EvmNetwork,
     /// The configuration for operations on the client.
-    ///
-    /// This will be shared across all clones of the client.
-    operation_config: Arc<ClientOperationConfig>,
+    config: ClientOperatingStrategy,
```

Comment:
Removed all the `Arc`s:
- `mpsc::Sender`s are MADE TO BE CLONED
- `EvmNetwork` and `ClientOperatingStrategy` are simpler to access like this and reasonably sized. As _premature optimization is the root of all evil_, I suggest we simplify this like that.